server/db/models/user.js

```diff
@@ -24,7 +24,11 @@ var schema = new mongoose.Schema({
     },
     google: {
         id: String
-    }
+    },
+    screenplay: [{
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
I believe the brackets go around just the value of type.  For example:
type: [mongoose.Schema.Types.ObjectId],
http://mongoosejs.com/docs/schematypes.html